compiler/rustc_middle/src/query/mod.rs

```diff
@@ -2293,6 +2293,17 @@ rustc_queries! {
         desc { "whether the item should be made inlinable across crates" }
         separate_provide_extern
     }
+
+    query call_site_abi_missing_features(key: (Ty<'tcx>, ty::InstanceKind<'tcx>)) -> &'tcx Vec<String> {
```

Comment:
We have a `List` type that you should probably use here.

(But that's not going to be making much of a perf difference.)

Reply:
`List<T>` seems to require `T: Copy`, which `String` isn't, and using a Symbol seems somewhat tricky (as I don't think it plays well with disk reading/writing, or at least I can't see other `cache_on_disk_if` queries that use it).
So I kept Vec.

Reply:
Why would `Symbol` not work well with queries cached on the disk? `Symbol` only needs `SpanDecoder`, which `CacheDecoder` does implement.